core/arxiv/submission/services/plaintext/plaintext.py

```diff
@@ -0,0 +1,279 @@
+"""
```

Comment:
Why not put in `__init__.py`? `import arxiv.submission.services.plaintext.plaintext` seems clumsy.

Reply:
See https://github.com/arXiv/arxiv-submission-core/pull/47#discussion_r248614236